frontend/src/firebase/actions.ts

```diff
@@ -475,6 +476,17 @@ export const sendInvite = async (
   await database.pendingInvitesCollection().add(newInvitation);
 };
 
+export const addUserInfo = async (email: string, fullName: string): Promise<void> => {
+  const userInfo: FirestoreUserData = {
+    name: fullName,
+  };
+  const userDoc = await database.usersCollection().doc(email);
+  const snapshot = await userDoc.get();
+  // since user info is only a name, we don't update if the user's current info already exists
+  if (!snapshot.exists) {
+    await userDoc.set(userInfo);
+  }
+};
```

Comment:
Might be good to do this in a transaction like

https://github.com/cornell-dti/office-hours/blob/06451761a6d68a0501f1ec41f53e46253513ecf5/src/firebasefunctions.ts#L23-L46

Reply:
Done.